src/misc/debounce2.ts

```diff
@@ -0,0 +1,40 @@
+// The MIT License (MIT)
```

Comment:
why not src/misc/debounce.ts ?

Reply:
I'm not sure but I guess the CI will probably just refuse to have both ts and js at the same time for the parser. And we still use ngeoDebounce everywhere else.